staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -36,6 +36,15 @@ type CostEstimator struct {
 
 func (l *CostEstimator) CallCost(function, overloadId string, args []ref.Val, result ref.Val) *uint64 {
 	switch function {
+	case "check":
+		// An authorization check has a fixed cost
+		// of 1/10th the typical pre-expression runtime cost limit
+		cost := uint64(100000)
```

Comment:
I think we should limit the number of authz calls, at least to start

something like ~2 max per expression, ~10 per policy?